hub/modules/config/configurations.bal

```diff
@@ -107,6 +107,11 @@ public configurable string CONSOLIDATOR_HEALTH_ENDPOINT = "/consolidator/actuato
 # Key for encryption and decryption of the hubsecret 
 public configurable string HUB_SECRET_ENCRYPTION_KEY = "g8caskkhrpvrp05l";
 
+# Below config will allow base64-encoded-bytes / alpha-numeric.
+# Recommended to use base64-encoded-bytes since alpha-numeric is considered less secure. 
+# This is just given to ensure the backward compatiblity
+public configurable string HUB_SECRET_ENCRYPTION_KEY_FORMAT = "alpha-numeric";
```

Comment:
default into the code lets use base64-encoded-bytes, in the external config we can use the alpha-numeric

Reply:
Changed the default configuration values as suggested.